scripts/aro-hcp/gen.sh

```
@@ -173,10 +207,34 @@ TEMPLATE_FILE_IS=$(dirname $0)/is-template.yaml
+        envsubst  < $TEMPLATE_FILE_ARO_ROLEASSIGNMENTS >> "$GEN_OUTPUT/aro.yaml"
+    else
+        echo appending identity references to: "$GEN_OUTPUT/aro.yaml" "(detach-on-delete, MSI from $MSI_RESOURCEGROUPNAME)"
+        envsubst  < $TEMPLATE_FILE_ARO_IDENTITIES_REF >> "$GEN_OUTPUT/aro.yaml"
+        echo appending role assignment references to: "$GEN_OUTPUT/aro.yaml" "(detach-on-delete)"
+        envsubst  < $TEMPLATE_FILE_ARO_ROLEASSIGNMENTS_REF >> "$GEN_OUTPUT/aro.yaml"
+    fi
 else
     echo creating: "$GEN_OUTPUT/is.yaml"
     envsubst  < $TEMPLATE_FILE_IS > "$GEN_OUTPUT/is.yaml"
 
+    TEMPLATE_FILE_IS_ROLES=$(dirname $0)/is-template-roles.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)